validator/validate.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import json
+import pyarrow.parquet as pq
+
+from jsonschema.validators import Draft7Validator
+
+
+def validate_geoparquet(input_file):
+    with open(os.path.join(sys.path[0], "schema.json"), "r") as f:
+        schema = json.loads(f.read())
```

Comment:
It doesn't look like you use the schema object after here, so I'd suggest dedenting all following lines for clarity. Also, why not use `json.load(f)` instead of `json.loads(f.read())`?

Reply:
Sure. Thanks!